### DISCUSSION TITLE
Port for switch homebrew possible?

### DISCUSSION THREAD

**Discussion by huntertrainboy** [OpenLoco/OpenLoco#1502] [Ideas]
I've been playing the current release for openrct2 on the switch homebrew platform. I was curious if these code bases are compatible enough for there to be a port of OpenLoco?

**Comment by AaronVanGeffen** [comment]
Not until we've completely reimplemented the game in C++. 

Currently, we still interface with the original game for the routines we haven't yet reimplemented. That means we can only generate (Intel) x86 binaries, not ARM ones the Switch would require. 

So not yet, but in time. Probably years.
